index.html

```diff
@@ -32,12 +32,12 @@ <h4>Relax and Enjoy the world of <span>Men's Fashion</span></h4>
         <div>
             <h1 class='center'><b>HATS</b></h1>
         </div>
-        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sint consequatur dicta iure eligendi quibusdam omnis
+        <p>Lorem ipsum dolor sit ame consectetur adipisicing elit. Sint consequatur dicta iure eligendi quibusdam omnis
             vero consequuntur dolores? Blanditiis consectetur quia aliquam, iusto vitae est numquam tempore cupiditate
             tenetur iure!</p>
         <img src="https://cdn.thegentlemansjournal.com/wp-content/uploads/2015/02/hat-TGJ.00.jpg">
```

Comment:
Please make sure img tags have "alt" attributes attached

Reply:
- [ ] Kindly note to always includes an explanation, a "why", for each requested change. For example:

This is a well-explained request:
- Please make sure `img` tags have "alt" attributes attached. The required `alt` attribute specifies an alternate text for an image, if the image cannot be displayed.
This request lacks a "why":
- I think it would be better to use the semantic tag: